### PR TITLE
Update config for internal r81 release (1.0.0)

### PR DESCRIPTION
**Pull request by gouthamve**
Signed-off-by: Goutham Veeramachaneni <gouthamve@gmail.com>

### CODE REVIEW

File: cortex/config.libsonnet [hunk 1]
```diff
@@ -165,18 +158,14 @@
...
         }
       else {}
     ),
 
     ringConfig: {
       'consul.hostname': 'consul.%s.svc.cluster.local:8500' % $._config.namespace,
-      'consul.consistent-reads': false,
       'ring.prefix': '',
-      'consul.watch-rate-limit': 1,
-      'consul.watch-burst-size': 5,
```
Comment (sandeepsukhani) [inline cortex/config.libsonnet:179]: The new default is now 1, is this change intentional?

Comment (gouthamve) [reply cortex/config.libsonnet:179]: Yes, it has been running in several prod envs without issues :) Its just the burst size, and if we are doing a qps more than 1, it'll be pretty quickly filled up anyways.
